core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.ts

```diff
@@ -379,6 +394,13 @@ export class ContributionsAndReview
   }
 
   getActiveDropdownTabChoice(): string {
+    if (this.activeTabType === this.TAB_TYPE_ACCOMPLISHMENTS) {
+      if (this.activeAccomplishmentType === 'stats') {
+        return 'Contribution Stats';
+      }
+      return 'Badges';
+    }
+
     if (this.activeTabType === this.TAB_TYPE_REVIEWS) {
```

Comment:
else if (and move to previous line)

Better still, construct a declarative dict so that you can do `return tabTypesToNames[this.activeTabType][subType];` -- that might be simpler to read.